lib/shoryuken/client.rb

```diff
@@ -9,15 +9,15 @@ def queues(name)
       end
 
       def sns
-        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns_endpoint))
+        @sns ||= Aws::SNS::Client.new(aws_client_options(:sns))
       end
 
       def sns_arn
         @sns_arn ||= SnsArn
       end
 
       def sqs
-        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs_endpoint))
+        @sqs ||= Aws::SQS::Client.new(aws_client_options(:sqs))
```

Comment:
Perhaps the AWS logging is a complex interaction of the gems used in my project, I am unsure, but I don't have an obvious answer.